zcash_primitives/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this library adheres to Rust's notion of
 
 ## [Unreleased]
 
+## [0.20.0] - 2024-11-14
```

Comment:
All of the downstream changelogs need to mention the migration to `zcash_primitives 0.20`. See e.g. the `zcash_primitives 0.19.0` release commit: c36c40818186484f435057019037f2fbc99fe705

Reply:
@nuttycom has informed me that for this particular release, these are being added in the downstream crate releases. I would prefer we don't do that unless we're using automation to ensure we aren't missing any, but given that this release train has already been prepared, I'm not going to block on changing it.